include/zephyr/sys/time.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024 Bjarki Arge Andreasen
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_SYS_TIME_H_
+#define ZEPHYR_INCLUDE_SYS_TIME_H_
+
+#include <zephyr/types.h>
+#include <zephyr/drivers/rtc.h>
+
+/** Get universal coordinated time timestamp */
+int sys_time_get_timestamp(int64_t *timestamp);
+/** Set universal coordinated time timestamp */
+int sys_time_set_timestamp(const int64_t *timestamp);
+
+/** Get universal coordinated time datetime */
+int sys_time_get_datetime(struct rtc_time *datetime);
+/** Set universal coordinated time datetime */
+int sys_time_set_datetime(const struct rtc_time *datetime);
+
+/** Convert universal coordinated time datetime to timestamp */
+int sys_time_datetime_to_timestamp(int64_t *timestamp, const struct rtc_time *datetime);
```

Comment:
Please make it clear that we're dealing with Unix timestamps here, as the timestamp name is also highly fraught in Zepyhr already.

Reply:
I will update the comments to specify unix time